app/views/teachers/sessions/check_email.html.erb

```diff
@@ -3,7 +3,16 @@
 <h1 class="govuk-heading-xl">Check your email</h1>
 
 <p class="govuk-body-m">
-  You should have received a link by email. Follow the link confirm your email address.
+  <% if @email.present? %>
+  You asked us to send a link to <%=@email%>.
+<% end %>
+</p>
+
+<p class="govuk-body-m">
+
+  If we have a record of that email address, we’ll send you a link to sign in.
+
+  Follow the link to confirm your email address.
```

Comment:
I think the content doesn't look quite right here, I imagine it would be something more like:

```suggestion
  <% if @email.present? %>
    You asked us to send a link to <%= @email %>. If we have a record of that email address, we’ll send you a link to sign in.
  <% else %>
    You should have received a link by email. Follow the link confirm your email address.
  <% end %>
```

Reply:
Good point, its repeating itself.